src/cdk/drag-drop/drag-ref.ts

```diff
@@ -93,10 +93,10 @@ export class DragRef<T = any> {
   private _pickupPositionOnPage: Point;
 
   /**
-   * Reference to the element that comes after the draggable in the DOM, at the time
-   * it was picked up. Used for restoring its initial position when it's dropped.
+   * Anchor node used to save the place in the DOM where the element was
+   * picked up so that it can be restored at the end of the drag sequence.
    */
-  private _nextSibling: Node | null;
+  private _anchor: Comment;
```

Comment:
Could we do `_placeholder` instead of `_anchor` to avoid overload with `<a>`?

Reply:
We can't, because there's something different that we're calling a `_placeholder` already (the element shown in the place of the one you're dragging).